install/helm/open-match/values.yaml

```diff
@@ -31,67 +31,82 @@ openmatch:
       endpoint: "/zipkin"
       reporterEndpoint: "zipkin"
     reportingPeriod: "5s"
-  backend:
+  synchronizer:
     install: true
-    replicas: 3
+    replicas: 1
     grpc:
-      port: 50505
+      port: 50501
     http:
-      port: 51505
+      port: 51501
     porttype: LoadBalancer
-  frontend:
+  backend:
```

Comment:
Why are all these port numbers changing?

Reply:
Just wanna make sure we are defining the port sequentially in this yaml file, we have a bunch of different components defined in this file and it is unclear what port we are not using when we wanna add something new if things are defined without any orders. The other point of this change is to use port with different postfix for the demo and e2e components (e.g 5050x for open match core components, 5051x for demo components and 5052x for e2e stuffs.)

Reply:
This will break cmd/swaggerui